docs/docs/guides/developer-guide/channel-aware/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Channelaware entities"
+showtoc: true
+---
+
+# Defining channel-aware entities
+
+Making an entity channel-aware means that it can be associated with a specific [Channel](/reference/typescript-api/channel/).
+This is useful when you want to have different data or features for different channels. First you will have to create
+a base entity ([Define a database entity](/guides/developer-guide/database-entity/)) that implements the `ChannelAware` interface.
```

Comment:
I would just say "an entity" rather than "a base entity", because the latter makes me think are are doing some inheritance.